script.js

```diff
@@ -174,10 +174,17 @@ function getLevel(job) {
 
 }
 function getLanguages(job) {
-    const languagesTagEl = document.createElement("div");
-    languagesTagEl.className = "jobDeatilTag"
-    languagesTagEl.innerHTML = `${job.languages}`;
-    return languagesTagEl;
+
+    for (const [language, value] of Object.entries(job.languages)) {
```

Comment:
selle koodiga saan hetkel sellise tulemuse

<img width="152" alt="Screenshot 2022-10-14 at 15 43 23" src="https://user-images.githubusercontent.com/101389533/195849772-625f9faa-0552-4c39-b01e-8d01e368ce24.png">

tundub köige lähemal, mis ma olnud olen, aga samas mötlen, et kui arvesse võtta sinu hommikust vihjet, siis kui mapida läbi languages key ja siis teha foreach igaühele ja luua iga kohta see tag... hmmmmm......

Reply:
miks sa ei saa teha `job.languages.map`? copy-paste mulle, mida tagastab `console.log(job.languages)`

Reply:
ja mida see pilt tähendab :D ?

Reply:
aaaaa sain aru pildist

Reply:
kui sa vaatad edasi mo pushe, siis näed praegust versiooni :D

Reply:
Sul on lihtsalt üks array, arrayga läbikäimine ei sõltu kuidagi sellest, kust see array tuli. See on samasugune array nagu todo listis oli sul tasks. Logi välja  `console.log(languagesForJobListing) ` ja sa näed, see tagastab lihstalt stringidest array. 

Kui sul oleks vaja kuvada iga array elemendi kohta <li/> element

ehk sul on array 

```
const array = ['esimene', 'teine', 'kolmas']
```


ja htmli peab tekkima

```
<li>esimene</li>
<li>teine</li>
<li>kolmas</li>
```

kas sa endiselt alustaksid kirjutades `for (const [order] of Object.entries(array))` ? 
kuidas sa kuvaks sellised elemendid?